src/state.cpp

```diff
@@ -203,4 +203,30 @@ int GetStateRate(int state_id, int rate) {
 
 }
 
+const std::vector<lcf::rpg::State*> GetObjects(const StateVec& states) {
+	std::vector<lcf::rpg::State*> state_list;
+	for (size_t i = 0; i < states.size(); ++i) {
+		if (states[i] > 0) {
+			lcf::rpg::State* state = lcf::ReaderUtil::GetElement(lcf::Data::states, i + 1);
+			state_list.push_back(state);
+		}
+	}
+	return state_list;
+}
+
+const std::vector<lcf::rpg::State*> SortedByPriority(const std::vector<lcf::rpg::State*>& states) {
+	std::vector<lcf::rpg::State*> state_list;
+	for (size_t i = 0; i < states.size(); ++i) {
+		state_list.push_back(states[i]);
+	}
```

Comment:
This is a copy: ``state_list = states``.

Reply:
Fixed.